### ISSUE TITLE
add to bottom of published and draft record pages in tiny print who created (owner = draft.parent.access.owned_by[0].resolve() ) the record

### ISSUE THREAD

**Issue by zguillen** [MSD-LIVE/issues#110]

**Comment by zguillen** [comment]
Can add to bottom of landing page while doing this related task https://github.com/MSD-LIVE/issues/issues/17

**Comment by jugovimm** [comment]
@zguillen I am trying to determine the location we want the text. By "published and draft record pages" do you mean these pages?

![Image](https://github.com/MSD-LIVE/issues/assets/144269927/f03d93c8-4224-4071-80cd-bc0870230b30)
![Image](https://github.com/MSD-LIVE/issues/assets/144269927/7cd39ff0-9d58-4387-a750-728f8855115b)

I can see in the first one where we could add who created the record:

![Image](https://github.com/MSD-LIVE/issues/assets/144269927/3c2c7a81-8022-4835-98d6-7f5a1102d008)

The second already has who created the record in the bottom right (unless by "draft record" you mean the page below?)

![Image](https://github.com/MSD-LIVE/issues/assets/144269927/0706c328-093f-4f47-977f-215703c4510c)

**Comment by zguillen** [comment]
yes, by draft record I mean that last screen shot. And agree thats a great spot to put it for the published record landing page.
